sentry/sentry-symfony/3.0/config/packages/sentry.yaml

```diff
@@ -0,0 +1,23 @@
+when@prod:
+    sentry:
+        dsn: '%env(SENTRY_DSN)%'
+        options:
+            excluded_exceptions:
+                - Symfony\Component\HttpKernel\Exception\NotFoundHttpException
+                - Symfony\Component\Security\Core\Exception\AccessDeniedException
+
+        # If you are using Monolog, you also need these additional configuration and services to log the errors correctly:
+        # https://docs.sentry.io/platforms/php/guides/symfony/#monolog-integration
+        # register_error_listener: false
+
+        # monolog:
+        #    handlers:
+        #        sentry:
+        #            type: service
+        #            id: Sentry\Monolog\Handler
+
+        # services:
+        #    Sentry\Monolog\Handler:
+        #        arguments:
+        #           $hub: '@Sentry\State\HubInterface'
+        #            $level: !php/const Monolog\Logger::ERROR
```

Comment:
please ensure that the indentation is always 4 spaces in all examples (with no leading space for the first line of yaml)